issue1296.p4(7): [--Werror=type-error] error: test_extern: contains self 'test_extern' as type argument
test_extern<test_extern<bit<32>>>() test;
^^^^^^^^^^^
issue1296.p4(3)
extern test_extern<T> {
       ^^^^^^^^^^^
issue1296.p4(13): [--Werror=type-error] error: test_extern1: contains self 'test_extern1' as type argument
test_extern1<test_extern<test_extern1<bit<32>>>>() test1;
^^^^^^^^^^^^
issue1296.p4(9)
extern test_extern1<T> {
       ^^^^^^^^^^^^
